metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/v3/OpenAPIV3Generator.java

```diff
@@ -1344,6 +1363,131 @@ private static Schema buildEntitySchema(
         .properties(properties);
   }
 
+  private static Schema buildSortSchema() {
+    // SortCriterion (inlined)
+    Schema sortCriterion =
+        newSchema()
+            .type(TYPE_OBJECT)
+            .description("Sort criterion: field + direction.")
+            .addProperties(
+                "field", newSchema().type(TYPE_STRING).description("Field name to sort by."))
+            .addProperties(
+                "order",
+                newSchema()
+                    .type(TYPE_STRING)
+                    ._enum(Arrays.asList("ASCENDING", "DESCENDING"))
+                    ._default("ASCENDING")
+                    .description("Sort order (default ASCENDING)."))
+            .addRequiredItem("field")
+            .addRequiredItem("order");
+
+    // Sort (top-level)
+    return newSchema()
+        .type(TYPE_OBJECT)
+        .description("Sort specification made up of one or more SortCriterion in evaluation order.")
+        .addProperties(
+            "sortCriteria",
+            newSchema()
+                .type(TYPE_ARRAY)
+                .description("List of sort criteria; applied in order.")
+                .items(sortCriterion)
+                .minItems(1))
+        .addRequiredItem("sortCriteria");
+  }
+
+  private static Schema buildFilterSchema() {
+    // Criterion schema
+    Schema criterion =
+        newSchema()
+            .type("object")
+            .description("A single field condition.")
+            .addProperties(
+                "field", newSchema().type(TYPE_STRING).description("The name of the field"))
+            .addProperties(
+                "value",
```

Comment:
We shouldn't expose "value" its been deprecated but exists in the data model for legacy reasons. Any new API should not expose this.